app/assets/stylesheets/application.css.scss

```diff
@@ -62,12 +62,15 @@ input {
   border: 0px;
 }
 
-.navbar-default .navbar-nav > li > a {
- color: white; /*Change active text color here*/
+.navbar-form > input[type="text"] {
+  width: 200px;
+  height: 30px;
+  border-radius: 4px;
+  border: 0px;
 }
 
-.navbar-default .navbar-nav > li > form {
- padding-top: 10px;
+.navbar-default .navbar-nav > li > a {
+ color: white; /*Change active text color here*/
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Use `//` comments everywhere<br>Line should be indented 2 spaces, but was indented 1 spaces